fleetspeak_python/fleetspeak/server_connector/connector.py

```diff
@@ -40,6 +42,45 @@
                     "The address to find the fleetspeak admin server, e.g. "
                     "'localhost:8080'")
 
+DEFAULT_TIMEOUT_SEC = 30
+
+_T = TypeVar("_T")
+
+
+# TODO: Remove retry logic when possible. I.e. when grpc supports it
+# natively - https://github.com/grpc/proposal/blob/master/A6-client-retries.md
+def RetryLoop(func: Callable[[float], _T],
+              timeout: Optional[float] = None,
+              single_try_timeout: Optional[float] = None) -> _T:
```

Comment:
Ditto: prefer to use `datetime.timedelta` rather than values with error-prone units hidden in the docs.

Reply:
Done.